src/gui/logbrowser.cpp

```diff
@@ -104,24 +112,46 @@ LogBrowser::~LogBrowser()
 {
 }
 
-void LogBrowser::closeEvent(QCloseEvent *)
+void LogBrowser::setupLoggingFromConfig()
 {
-    ConfigFile cfg;
-    cfg.saveGeometry(this);
+    ConfigFile config;
+    auto logger = Logger::instance();
+
+    if (config.automaticLogDir()) {
+        // Don't override other configured logging
+        if (logger->isLoggingToFile())
+            return;
+
+        logger->setupTemporaryFolderLogDir();
+        if (auto deleteOldLogsHours = config.automaticDeleteOldLogsAge()) {
+            logger->setLogExpire(deleteOldLogsHours->count());
```

Comment:
Too bad that setLogExpire also is not taking a std::chrono arg.  
now, std::chrono does not help because we do not see that it is in hours.